dashel/dashel-posix.cpp

```diff
@@ -1078,6 +1078,13 @@ namespace Dashel
 				ioctl(fd, TIOCMBIS, &iFlags);
 			else
 				ioctl(fd, TIOCMBIC, &iFlags);
+			
+            //Ignore SIGHUP from this point ownward.
```

Comment:
please use tabs instead of space

Reply:
Also, please give a brief explanation why, suck as "we do not want the user program to stop if there is a disconnection on a serial port".